src/main/java/no/unit/nva/model/contexttypes/event/ReifiedExtent.java

```diff
@@ -0,0 +1,42 @@
+package no.unit.nva.model.contexttypes.event;
+
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+import java.util.Objects;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+public class ReifiedExtent {
```

Comment:
Is this an abstract representation of duration, e.g. 3 generations, 4 aeons, 2 lifetimes etc.?